inst/scripts/make-hpca-metadata.R

```diff
@@ -0,0 +1,27 @@
+write.csv(file="../extdata/metadata-hpca.csv", 
+          data.frame(
+              Title = sprintf("Human Primary Cell Atlas data %s", c("normcounts", "colData")), # This can be the exact file name (if self-describing) or a more complete description.
+              Description = sprintf("%s human primary cell atlas data with 38 main cell types (169 subtypes)", 
+                                    c("Matrix or norm. expression", "Per-sample metadata: cell type labels")),
```

Comment:
The `Description` should be a full english sentence, so the format string should be something like:

- Matrix of normalized expression values from the Human Primary Cell ...
- Per-sample metadata containing cell type labels from the Human ...